batch_job/src/batch_job_mesos.c

```diff
@@ -68,7 +76,13 @@ static void start_mesos_scheduler(struct batch_queue *q)
 	path_dirname(exe_path, exe_dir_path);
 
     char *exe_py_path = string_format("%s/mf_mesos_scheduler", exe_dir_path);
-	char *envs[] = {"LD_PRELOAD=/afs/nd.edu/user37/ccl/software/external/gcc-4.9.3/amd64_linux26/lib64/libstdc++.so.6:/afs/nd.edu/user37/ccl/software/external/svn-1.9.4/amd64_linux26/lib/libsvn_delta-1.so", "CCTOOLS=/afs/crc.nd.edu/user/c/czheng2/cctools", NULL};
+	char *ld_preload_str = NULL;
+
+	if(mesos_preload != NULL) {
+		ld_preload_str = string_format("LD_PRELOAD=%s", mesos_preload);
+	} 
+	// TODO: Environment variables set for launching Mesos, this would not work outside ND. Ask user to specify the environments?
+	char *envs[] = {"CCTOOLS=/afs/crc.nd.edu/user/c/czheng2/cctools", ld_preload_str, NULL};
```

Comment:
What is the proper way to pass `cctools` path to makeflow or batch job system? To run makeflow with Mesos, we must know the path to `cctools` binary. @dthain @btovar